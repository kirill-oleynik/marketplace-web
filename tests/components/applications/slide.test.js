const React = require('react');
const { shallow } = require('enzyme');
const toJSON = require('enzyme-to-json').default;
const ApplicationSlide = require('../../../app/components/applications/slide').default;

describe('#render', () => {
  test('it renders correctly', () => {
    const component = shallow(
      <ApplicationSlide
        application={{
          id: 1,
          title: 'Test1',
          description: 'Hello World!',
          logo: 'https://example.com/image.png'
        }}
      />
    );

    expect(
      toJSON(component)
    ).toMatchSnapshot();
  });
});
