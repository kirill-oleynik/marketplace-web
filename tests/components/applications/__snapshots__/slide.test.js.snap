// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#render it renders correctly 1`] = `
<div
  className="carousel__item  "
>
  <div
    className="carousel__item-content"
  >
    <span
      className="app-item__img mb-5"
      style={
        Object {
          "backgroundImage": "url('https://example.com/image.png')",
        }
      }
    />
    <p
      className="app-item-slide__title font-16 font-700 mb-5"
    >
      Test1
    </p>
    <p
      className="app-item-slide__descr"
    >
      Hello World!
    </p>
  </div>
</div>
`;
